media-tv/kodi/kodi-9999.ebuild

```diff
@@ -133,6 +133,11 @@ DEPEND="${COMMON_DEPEND}
 # generated addons package.  #488118
 [[ ${PV} == "9999" ]] && DEPEND+=" virtual/jre"
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-9999-no-arm-flags.patch #400618887
+	"${FILESDIR}"/${PN}-9999-texturepacker.patch
+)
```

Comment:
other ebuilds do so, because they do not override src_prepare(). It's a good practice to minimize the scope of vars as much as possible.